docs/RFCs/0009-cross-language-logging-integration.md

```diff
@@ -0,0 +1,210 @@
+# RFC: Cross-Language Logging Integration for libdatadog
```

Comment:
Given that we may want to clarify the RFC a bit, do you maybe want to add this in a separate PR? (Or add it now and change it later, up to you)